src/main/java/org/apache/commons/compress/archivers/tar/TarArchiveEntry.java

```diff
@@ -553,6 +554,23 @@ public TarArchiveEntry(final byte[] headerBuf, final ZipEncoding encoding, final
         parseTarHeader(headerBuf, encoding, false, lenient);
     }
 
+    /**
+     * Construct an entry from an archive's header bytes for random access tar. File is set to null.
+     * @param headerBuf The header bytes from a tar archive entry.
+     * @param encoding encoding to use for file names
+     * @param lenient when set to true illegal values for group/userid, mode, device numbers and timestamp will be
+     * ignored and the fields set to {@link #UNKNOWN}. When set to false such illegal fields cause an exception instead.
+     * @param dataOffset Position of the entry data in the random access file
```

Comment:
Formatting is inconsistent. Some descriptions start with an upper case letter, others not. Some end with a period, others not.  But it's a nit-pick, and can be fixed later :+1:

Reply:
I formatted all the added Javadoc in `TarArchiveEntry` to start with lower case letter and end with a period.